src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocpackage/legacywithboth/Example3.java

```diff
@@ -0,0 +1,14 @@
+/*xml
+<module name="Checker">
+  <module name="JavadocPackage">
+    <property name="allowLegacy" value="true"/>
+  </module>
+</module>
+*/
+// xdoc section -- start
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocpackage.legacywithboth;
+// violation 9 lines above 'Legacy package.html file should be removed'
```

Comment:
in html, line `9 lines above` looks weird, especially as user see "package ..." it is usual first line in java file.

lets do a trick: 
```
// xdoc section -- start
package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocpackage.legacywithboth;
/* violation on first line 'Legacy package.html file should be removed' */
// xdoc section -- end` right after "package"
// violation 11 lines above 'Legacy package.html file should be removed'
.....
```

so we remove violation line from section that we print in html and keep multiline comment that is not recognized as special comment, so it is plain text, we can write anything we want.

Reply:
Done